common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -157,6 +156,18 @@ def get_course_social_sharing_widget(self, widget_name):
         """ Retrieves the specified social sharing widget by its classification """
         return self.q(css='a.action-{}'.format(widget_name))
 
+    def get_courses(self):
+        """
+        Get all courses shown in the dashboard
+        """
+        return self.q(css='ul.listing-courses .course-item')
+
+    def get_course_date(self):
```

Comment:
If we have multiple courses on student dashboard which course's start date it would return?